pages/linux/ydotool.md

```diff
@@ -0,0 +1,16 @@
+# ydotool
+
+> Display server agnostic input automator.
```

Comment:
Isn't this software more of a keyboard "emulator" for entering keys and such through commands?

If so it may be better to communicate this to the user.

```suggestion
> Control keyboard and mouse inputs via commands.
```

Reply:
I feel like "Display server agnostic" is kind of important, because `ydotool` derives its name from `xdotool`, which specifically targets X11.

Reply:
Yeah, it's the reason why I chose that wording. If there was no succession line, I would have chosen a different description.

Reply:
I only suggested it since it makes the most sense to me, I have no idea what a server agnostic is. Can we come up a wording which makes sense to both newbs and people who the software.

Reply:
"Display server" agnostic. `xdotool` connects to X11 directly to input keys and mouse events whereas `ydotool` creates a virtual input device. Your comment gave me an idea how to reword this. Just a moment.